src/view/components/common/modal-dialog/dialog-component.ts

```diff
@@ -49,6 +50,10 @@ export class ModalDialogComponent {
       } else {
         this.removeEscapeListener()
       }
+
+      if (!change.hidden.currentValue){
```

Comment:
if(!this.hidden) ...

change.foo.currentValue will always be equal to this.foo. Not a big deal.